api/api/src/main/scala/uk/ac/wellcome/platform/api/services/WorksRequestBuilder.scala

```diff
@@ -66,6 +66,12 @@ object WorksRequestBuilder
         .field("data.subjects.label.keyword")
         .minDocCount(0)
 
+    case AggregationRequest.Contributor =>
+      TermsAggregation("contributors")
+        .size(20)
```

Comment:
Does 20 include all the contributors we have?

Reply:
I chose this the same as `subject` and `genre`. I have no idea tbh. Maybe @jamesgorrie has some idea?

Reply:
For some context on number of aggregations, this comment:
https://github.com/wellcomecollection/platform/issues/3944#issuecomment-547065812